packages/cozy-client/src/store.spec.js

```diff
@@ -216,7 +216,7 @@ describe('Store', () => {
       expect(queryResult.data.map(x => x._id)).toEqual(['todo_1', 'todo_2'])
       queryResult = getQueryFromStore(store, 'Q2')
       expect(queryResult.data.length).toBe(2)
-      expect(queryResult.data.map(x => x._id)).toEqual(['todo_3', 'todo_4'])
+      expect(queryResult.data.map(x => x._id)).toEqual(['todo_1', 'todo_2'])
```

Comment:
failing job : https://app.travis-ci.com/github/cozy/cozy-client/builds/255039439

i am not convinced about that change